analyzers/tests/SonarAnalyzer.TestFramework/Extensions/StringAssertionsExtensions.cs

```diff
@@ -28,6 +28,10 @@ public static class StringAssertionsExtensions
     public static void BeIgnoringLineEndings(this StringAssertions stringAssertions, string expected) =>
         stringAssertions.Subject.ToUnixLineEndings().Should().Be(expected.ToUnixLineEndings());
 
+    [CustomAssertion]
+    public static void BeIgnoringLineEndingsAndEmptyLines(this StringAssertions stringAssertions, string expected) =>
```

Comment:
This doesn't make much sense to me. I'd prefer to see empty lines in the asserted text.